### PR TITLE
fix: enable all parent pom plugins

### PR DESCRIPTION
**Pull request by octonato**
Parent poms were not working because we need to explicitly enable the plugins.

### CODE REVIEW

File: updateSdkVersions.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ updateScalaSamples() {
 
 updateMavenPlugin() {
   echo ">>> Updating maven plugin to $SDK_VERSION"
-  cd maven-java && mvn versions:set -DnewVersion="$SDK_VERSION" && cd ..
+  cd maven-java && mvn --activate-profiles patch-version versions:set -DnewVersion="$SDK_VERSION" && cd ..
```
Comment (octonato) [inline updateSdkVersions.sh:42]: hmm, we neeed to call the path-maven-version here as well.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -316,6 +316,7 @@ jobs:
       - name: Value Entity Archetype
         run: |-
           export SDK_VERSION=$(cat ~/kalix-sdk-version.txt)
```
Comment (ennru) [inline .github/workflows/ci.yml:318]: Is this version file still there? Alternatively, this should reference `steps.build_sdk.outputs.sdk_version`, right?

File: .github/publish-maven.sh [hunk 3]
```diff
@@ -75,4 +75,4 @@ echo "${PGP_SECRET}" | base64 -d | gpg --import --batch
 
 # Maven deploy with profile `release`
 # mvn --quiet --batch-mode --activate-profiles release deploy
-mvn --batch-mode --activate-profiles release deploy
+mvn --batch-mode --activate-profiles release -Dskip.docker=true deploy
```
[the remark this answers is not in the document]
Comment (ennru) [reply .github/publish-maven.sh:65]: I understand it is backward in the Docker plugin already. The negation can become confusing, but I don't see what we could do about that.

**Review by ennru — APPROVED** [review]
LGTM.

**Comment by octonato** [comment]
@ennru, I merged as is now and will check your suggestions next. I will send a follow-up PR to improve.

**Pull request merged**